src/com/tony/server/response/ResponseBuilder.java

```diff
@@ -21,27 +26,31 @@ public static String buildHeaderString(HashMap<String, String> headers) {
     }
 
     public static String buildHeaderLine(String key, String value) {
-        return key + ": " + value + "\n";
+        return key + HEADER_SEPARATOR + value + CRLF;
     }
 
     public static String buildBody(String body) {
         if((body == null) || (body.equals(""))){
             return "";
         } else{
-            return "\n" + body;
+            return CRLF + body;
         }
     }
 
-    public static byte[] buildImageResponse(String statusLine, HashMap<String, String> headers, byte[] imageBytes) {
+    public static byte[] buildImageResponse(Status status, HashMap<String, String> headers, byte[] imageBytes) {
         ByteArrayOutputStream baos = new ByteArrayOutputStream();
         try {
-            baos.write(statusLine.getBytes());
+            baos.write(buildStatusLine(status).getBytes());
             baos.write(buildHeaderString(headers).getBytes());
-            baos.write("\n".getBytes());
+            baos.write(CRLF.getBytes());
             baos.write(imageBytes);
         } catch (IOException e) {
             e.printStackTrace();
```

Comment:
While its not part of this diff, you have a logger now, and you should use it to log this.